packages/dataviews/src/view-table.js

```diff
@@ -242,37 +242,32 @@ function TableRow( {
 				'is-selected':
 					hasPossibleBulkAction && selection.includes( id ),
 				'is-hovered': isHovered,
+				'has-bulk-actions': hasPossibleBulkAction,
 			} ) }
 			onMouseEnter={ handleMouseEnter }
 			onMouseLeave={ handleMouseLeave }
-			onClickCapture={ ( event ) => {
```

Comment:
Yeah, the greedy `onClickCapture` approach was justifiable when it was predicated on `event.ctrlKey || event.metaKey`, otherwise it immediately becomes a blunt tool. Case in point, from the current branch:

https://github.com/WordPress/gutenberg/assets/150562/02c5c3bb-e322-4bdd-b25d-cbea55005cd6

There are tradeoffs to make here. Some approaches:

* Simplify, get rid of this trick, expand the title box area, and listen to regular (non captured) click events on the title box. Get rid of the fancy ctrl/meta click behaviour.
* Keep both side by side: ctrl/meta catches _any_ click on the row, ignoring any underlying elements, while regular clicks with no modifiers will be caught on the title box (but no wider). After very short consideration, this is what I'd try first.
* Go for the more complex solution without title boxes. For instance:
    * Ctrl/meta-clicks anywhere on the row should be caught at capture time, trigger a selection toggle, and stop event propagation.
    * Regular clicks anywhere on the row will need to consider whether more granular listeners were triggered (i.e. whether the user has clicked on any interactive element inside the row, like a button). There some ways to do this (e.g. inspecting all DOM nodes between `event.target` and `event.currentTarget` to find attached listeners), but I don't know how robust they are.